authors/apps/articles/models.py

```diff
@@ -1,7 +1,7 @@
 import uuid
-
 from django.db import models
 from taggit.managers import TaggableManager
+from django.utils.text import slugify
```

Comment:
I thought this was already on develop why include another one?

Reply:
Check line 6 of the same file

Reply:
I may have accepted changes wrongly when rebasing. Let me fix it right away.